### DISCUSSION TITLE
Which version? qt or non qt? Which is better? [Windows 11]

### DISCUSSION THREAD

**Discussion by CrossbowDemon** [smplayer-dev/smplayer#462] [Q&A] [answered]
What's the difference between the qt5.6 and non-qt5.6 version?
Does one or the other run better? Which one would be better on Windows 11?

smplayer-22.2.0-x64-qt5.6-unsigned.exe
or
smplayer-22.2.0-x64-unsigned.exe

I get they are probably extremely similar, but I can't stop thinking about it.

**Comment by smplayer-dev** [comment, marked as answer]
The qt5.6 package uses Qt 5.6 while the other package uses Qt 5.15.
The qt5.6 package is mainly for compatibility with older systems.
